Sources/KlaviyoUI/KlaviyoWebView/KlaviyoWebViewModel.swift

```diff
@@ -28,8 +28,16 @@ class KlaviyoWebViewModel: KlaviyoWebViewModeling {
         var scripts: [String: WKUserScript] = [:]
 
         if let closeHandlerScript = try? FileIO.getFileContents(path: "closeHandler", type: "js") {
-            let script = WKUserScript(source: closeHandlerScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
+            #if swift(<6.0)
+            let script = try WKUserScript(source: closeHandlerScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
             scripts["closeHandler"] = script
+
+            #else
+            Task {
+                let script = await WKUserScript(source: closeHandlerScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
+                scripts["closeHandler"] = script
+            }
```

Comment:
hmm, by dispatching this to a task, the `KlaviyoWebView` will read the `loadScripts` property before the `closeHandler` script has been loaded. I can think some more tomorrow about how to resolve this; it might involve some refactoring of the View/ViewModel